pages.it/common/git-commit.md

```diff
@@ -0,0 +1,20 @@
+# git commit
+
+> Salva file nell'area di stage in una nuova istantanea del tuo repository.
+> Maggiori informazioni: <https://git-scm.com/docs/git-commit>.
+
+- Committa sul repository i file nell'area di stage, insieme a un messaggio:
```

Comment:
"Committare" it's used as a verb also in the official manual (e.g., https://git-scm.com/book/it/v1/Basi-di-Git-Salvare-le-modifiche-sul-repository). However, I tried to use it as less as possible